package.json

```diff
@@ -571,7 +571,6 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "gulp build --mode=production",
```

Comment:
What was the reason we needed to move this step into the pipeline explicitly instead of relying on vsce to run this for us?

Reply:
Good catch. This is left over from when I was using vsce to repack the extension after extracting it, but now that that's changed to ArchiveFiles I should be able to remove these changes.